packages/web3wallet/src/types/engine.ts

```diff
@@ -38,10 +38,12 @@ export abstract class IWeb3WalletEngine {
   public abstract updateSession(params: {
     topic: string;
     namespaces: SessionTypes.Namespaces;
-  }): Promise<void>;
+  }): Promise<{ acknowledged: () => Promise<void> }>;
 
   // update session expiry (SIGN)
-  public abstract extendSession(params: { topic: string }): Promise<void>;
+  public abstract extendSession(params: {
+    topic: string;
+  }): Promise<{ acknowledged: () => Promise<void> }>;
```

Comment:
Will require a docs update right? 

Guess this can be seen as non-breaking since we're going from `void` to non-void return type

Reply:
https://github.com/WalletConnect/walletconnect-docs/pull/1511